### PR TITLE
Added docs on how to use helm charts for TID install

### PR DESCRIPTION
**Pull request by PetrMc**
Per multiple customer requests - having Helm Charts info should be helpful for customer interactions.

### CODE REVIEW

**Comment by PetrMc** [comment]
@pmerrison @batchamalick @nauticalmike @psbrar99 @ogarrett would appreciate your feedback - thanks!

File: content/en/helm/_index.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+```bash
+helm repo add tid https://tetratelabs.github.io/istio-helm/
+helm repo update tid
+```
+
+  Output:
+  
+  ```pre
+   $ helm repo add tid https://tetratelabs.github.io/istio-helm/
```
Comment (nauticalmike) [inline content/en/helm/_index.md:26]: I see now the chart repo is different, compared to [this one](https://github.com/tetratelabs/helm-charts/blob/main/README.md#setup-helm-repo), which one is the official we need to provide to our TID customers?

**Review by nauticalmike — COMMENTED** [review]
LGTM if this is going to be the official chart repo

File: content/en/helm/_index.md (hunk 1 above)
Comment (PetrMc) [reply content/en/helm/_index.md:26]: thanks @nauticalmike 
the repo which is mentioned in this PR can be updated automatically while the one you're linking to is semi-manual work.
as for `official` - if you know who decides - we can ask that person :-)

**Review by batchamalick — APPROVED** [review]
LGTM

**Comment by nauticalmike** [comment]
https://github.com/tetrateio/tid-internal/issues/73

**Pull request merged**